src/components/views/rooms/EventTile.tsx

```diff
@@ -576,6 +576,12 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
 
         const encryptionInfo = MatrixClientPeg.get().getEventEncryptionInfo(mxEvent);
         const senderId = mxEvent.getSender();
+        if (!senderId) {
+            // something definitely wrong is going on here
```

Comment:
(aside) what's going wrong is the js-sdk conflating room events with completely different things like account data and trying to stuff both things into the same type, so that `MatrixEvent.getSender` has to return an optional value. But that's a bigger problem. See also https://github.com/matrix-org/matrix-spec/issues/897.